app/src/layouts/tabular/index.ts

```diff
@@ -142,9 +142,8 @@ export default defineLayout<LayoutOptions, LayoutQuery>({
 		function useItemOptions() {
 			const page = syncRefProperty(layoutQuery, 'page', 1);
 			const limit = syncRefProperty(layoutQuery, 'limit', 25);
-			const sort = syncRefProperty(layoutQuery, 'sort', () =>
-				primaryKeyField.value ? [primaryKeyField.value?.field] : []
-			);
+			const defaultSort = computed(() => (primaryKeyField.value ? [primaryKeyField.value?.field] : []));
+			const sort = syncRefProperty(layoutQuery, 'sort', defaultSort.value);
```

Comment:
This is no longer reactive if my understanding of `.value` is correct. The `.value` here is the static value, while the `defaultSort` is the reactive object reference. Seeing you're using `unref` in the `syncRefProperty` util, I believe you can safely omit the `.value` from the usage here, and keep it reactive (eg make sure the default value updates when the `primaryKeyFIeld.value` updates)

Reply:
You're right, that was my intent! I hadn't push that change yet 😁